DemoqaProject/pageObjects/Elements.cs

```diff
@@ -7,10 +7,323 @@ namespace DemoqaProject.PageObjects
+        {
+            return fullNameInput.GetAttribute("placeholder");
+        }
+
+        public string GetEmailPlaceholder()
+        {
+            return emailInput.GetAttribute("placeholder");
+        }
+
+        public string GetCurrentAddPlaceholder()
+        {
+            return currentAddressInput.GetAttribute("placeholder");
+        }
+
+        public void NavigateToCheckBox()
+        {
+            checkBoxButton.Click();
+            selectAll.Click();
+        }
+
+        public void Count()
+        {
+            ReadOnlyCollection<IWebElement> checkbox = driver.FindElements(By.XPath("//span[@class='text-success']"));
+            Console.WriteLine(checkbox.Count);
+            if (checkbox.Count==17)
+            {
+                Console.WriteLine("Selected all");
+            }
+            else if (!selectAll.Selected)
+            {
+                Console.WriteLine("Deselected all");
+            }
+            selectAll.Click();
+        }
+
+        public void SpecificCategories()
+        {
+            //Select specific category and verify
+            arrowButton.Click();
+            if (!desktopButton.Selected)
+            {
+                desktopButton.Click();
+                Console.WriteLine("Category Desktop selected");
+            }
+            else
+            {
+                Console.WriteLine("Category Desktop not selected");
+            }
+        }
+
+        public void SelectGroupAndUncheckSub()
+        {
+            ReadOnlyCollection<IWebElement> checkbox = driver.FindElements(By.XPath("//button[@class='rct-collapse rct-collapse-btn']"));
+            checkbox[1].Click();
+            if (!commandsButton.Selected)
+            {
+                commandsButton.Click();
+                Console.WriteLine("Sub group deselected");
+            }
+        }
+
+        public void ExpendAndCollapseAll()
+        {
+            expendAll.Click();
+            collapseAll.Click();
+        }
+
+        public void NavigateToRadioButton()
+        {
+            radioButton.Click();
+        }
+
+        public void CheckYesRadio()
+        {
+            yesRadio.Click();
+        }
+        
+        public void CheckImpressiveRadio()
+        {
+            impressiveRadio.Click();
+        }
+
+        public void NavigateToWebTables(string firstname, string lastname, string email, string age, string salary, string department)
+        {
+            WebTablesButton.Click();
+            AddButton.Click();
+            firstNameInput.SendKeys(firstname);
+            lastNameInput.SendKeys(lastname);
+            emailInput.SendKeys(email);
+            ageInput.SendKeys(age);
+            salaryInput.SendKeys(salary);
+            departmentInput.SendKeys(department);
+            submitForm.Click();
+        }
+
+        public void EditRecord()
+        {
+            editRecord.Click();
+            firstNameInput.Clear();
+            firstNameInput.SendKeys("Tom");
```

Comment:
Do not hardcode this. Add param to method.

Reply:
done